ops-devnet/docker-compose.yml

```diff
@@ -30,7 +30,7 @@ services:
 
   l2:
     pid: host # allow debugging
-    image: kromanetwork/geth:dev
+    image: kromanetwork/geth
```

Comment:
Please modify this with a tag for the build that corresponds to the version of go-ethereum referenced in `go.mod`.